src/List/List.js

```diff
@@ -45,13 +49,16 @@ export default class List extends Component {
       className: classNameProp,
       component,
       padding,
+      children,
       ...other,
     } = this.props;
     const classes = this.context.styleManager.render(styleSheet);
+    const firstChild = Children.toArray(children)[0];
```

Comment:
Computationally speaking, we could be more efficient. I'm wondering if it worth having a
Children utility function.